.travis.yml

```diff
@@ -30,13 +30,17 @@ matrix:
 before_script:
   - composer selfupdate
   - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "5.3" ]]; then composer remove phpcr/phpcr-benchmarks --dev; fi
```

Comment:
i would propose to drop it in version 2, to also be able to use more
concise code thanks to the new constructs in 5.4.

i think skipping benchmarks on 5.3 is a perfectly valid solution for now.